model/src/channel/message/kind.rs

```diff
@@ -22,6 +22,8 @@ pub enum MessageType {
     ChannelFollowAdd = 12,
     GuildDiscoveryDisqualified = 14,
     GuildDiscoveryRequalified = 15,
+    /// Message is a inline reply
```

Comment:
I feel like I've fixed this before but my changes got overwritten, so I'll just comment it this time
```suggestion
    /// Message is an inline reply.
```